core/settings.py

```diff
@@ -192,3 +192,19 @@
     profiles_sample_rate=1.0,
     environment=ENV or "local",
 )
+
+AZURE_AUTH = {
+    "CLIENT_ID": os.environ.get("CLIENT_ID"),
+    "CLIENT_SECRET": os.environ.get("CLIENT_SECRET"),
+    "REDIRECT_URI": os.environ.get("REDIRECT_URI"),
+    "SCOPES": ["User.Read"],
+    "AUTHORITY": os.environ.get("AUTHORITY"),
+    "LOGOUT_URI": "https://localhost:8000/logout",
+}
+LOGIN_URL = "/azure_auth/login"
+LOGIN_REDIRECT_URL = "/search"  # Or any other endpoint
+
+AUTHENTICATION_BACKENDS = ("azure_auth.backends.AzureBackend",)
```

Comment:
I think we can probably swap this for an [AllowAllUsers backend](https://docs.djangoproject.com/en/5.0/ref/contrib/auth/#django.contrib.auth.backends.AllowAllUsersModelBackend) in tests

Reply:
This is conditional on the AZURE_AUTH_ENABLED now so we wont have a backend when we dont need it.